src/platform/packages/shared/response-ops/alerts-apis/constants.ts

```diff
@@ -8,15 +8,4 @@
  */
 
 export const BASE_ALERTING_API_PATH = '/api/alerting';
-
-export const queryKeys = {
```

Comment:
I moved all queryKeys and queryMutations of the new packages to dedicated files to increase visibility and because they aren't properly constants in the JS reference sense of the term